### PR TITLE
Add Sepolia WETH and Uni

### PR DESCRIPTION
**Pull request by cbachmeier**
Mimicking the style of the goerli file
[Sepolia WETH](https://sepolia.etherscan.io/token/0x7b79995e5f793a07bc00c21412e50ecae098e7f9)
[Sepolia Uni](https://sepolia.etherscan.io/address/0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984)

### CODE REVIEW

**Comment by cbachmeier** [comment]
Need to verify UNI address

**Comment by cbachmeier** [comment]
Confirmed UNI address is correct

**Review by matteenm — COMMENTED** [review]
also need to update the buildList.js file to pull in from this new file. here is a similar pr for BNB for reference: https://github.com/Uniswap/default-token-list/pull/1533/files

File: src/tokens/sepolia.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "Wrapped Ether",
+    "address": "0xfff9976782d46cc05630d1f6ebab18b2324d6b14",
```
Comment (matteenm) [inline src/tokens/sepolia.json:4]: lets use the checksummed address here

**Review by matteenm — APPROVED** [review]

**Pull request merged**